app/assess/templates/sub_criteria.html

```diff
@@ -27,8 +27,53 @@
         </p>
 
         {{ logout_partial(sso_logout_url) }}
+
     </div>
 
+    {% if current_theme_id == 'score' %}
```

Comment:
We aren't using this parameter anymore, I think this logic should be in the `score.html` e.g. https://github.com/communitiesuk/funding-service-design-assessment/blob/f9269f74c6728cb2348296dd58cea63ee7e3c1ff/app/assess/routes.py#L188-L202